templates/pricing/desktop.html

```diff
@@ -74,176 +71,326 @@ <h2 class="p-heading--4">Additional benefits for Enterprise Services Plus</h2>
-                <li>CUPS (printing)</li>
-                <li>
-                  <i>Additional third-party products can be installed and automatically configured depending on the requirements.</i>
-                </li>
-              </ul>
-            </td>
-          </tr>
-          <tr>
-            <th>CIS Benchmarking</th>
-            <td></td>
-            <td>
-              <p>CIS Benchmark configuration and support</p>
-            </td>
-          </tr>
-          <tr>
-            <th>Snap Store configuration</th>
-            <td></td>
-            <td>
-              <p>Snap Store proxy deployment and local snap package repository.</p>
-            </td>
-          </tr>
-          <tr>
-            <th>CIFS/SMB configuration</th>
-            <td></td>
-            <td>
-              <p>Workstations configured to mount CIFS/SMB file systems from a third-party source</p>
-            </td>
-          </tr>
-          <tr>
-            <th>Security Configuration</th>
-            <td>
-              <p>Installed software audit</p>
-              <p>Security policy enforcement</p>
-              <p>Anti-virus/anti-malware solution support</p>
-            </td>
-            <td>
-              <p>Installed software audit</p>
-              <p>Custom firewall enablement</p>
-              <p>Security policy enforcement</p>
-              <p>Anti-virus/anti-malware solution support</p>
-            </td>
-          </tr>
-          <tr>
-            <th>Encrypted Volume Support</th>
-            <td></td>
-            <td>
-              <p>Third party encryption support</p>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-      <p>
-        <a class="p-button"
-           href="https://assets.ubuntu.com/v1/a48fc9cd-Ubuntu%20Desktop%20DS%2021.06.23.pdf">Download the datasheet</a>
-        <a href="/desktop/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
-      </p>
+      <hr class="p-rule" />
+      <h2>Find the service for your enterprise</h2>
+    </div>
+    <div class="row--25-75">
+      <div class="col">
+        <table aria-label="Comparison of Ubuntu Desktop Enterprise Services">
+          <thead>
+            <tr>
+              <th role="presentation">&nbsp;</th>
+              <th scope="col">Ubuntu Desktop Enterprise Services</th>
+              <th scope="col">Ubuntu Desktop Enterprise Services Plus</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <th scope="row">Cost</th>
+              <td>
+                <strong>$50,000</strong>
+              </td>
+              <td>
+                <strong>$105,000</strong>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row" rowspan="2">Landscape configuration</th>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">
+                  Machine registration and monitoring via Landscape SaaS or on-prem
```

Comment:
Can we add line breaks here to match design?

Reply:
I can't see what lines are missing here

Reply:
```suggestion
                  Machine registration and monitoring <br />via Landscape SaaS or on-prem
```